generation/generate_release_please_config.sh

```diff
@@ -26,11 +26,10 @@ for path in $module_list; do
   module_line=$(grep -E "^((google-.*|grafeas|gapic\-libraries)).*:[0-9]+\.[0-9]+\.[0-9]+.*:[0-9]+\.[0-9]+\.[0-9]+.*$" "${version_file}")
 
   artifact_name=$(echo "${module_line}" | cut -d ":" -f1)
-  module_released_version=$(echo "${module_line}" | cut -d ":" -f2)
   module_snapshot_version=$(echo "${module_line}" | cut -d ":" -f3)
```

Comment:
Can you rename it to module_current_version? The 3rd column is marked as current.

Reply:
Done.